checkers-flex.html

```diff
@@ -87,15 +90,95 @@
     </div>
     <!--1st row -->
     <div class="row">
-      <div class="box black-square" id="a1" nextValidSquare="['b2']"></div>
+
+      <div class="box black-square" id="a1" nextValidSquare="['b2']">
+        <div class="piece black" :id="player1.chips[0].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
-      <div class="box black-square" id="c1" nextValidSquare="['b2', 'd2']"></div>
+
+      <div class="box black-square" id="c1" nextValidSquare="['b2', 'd2']">
+        <div class="piece black" :id="player1.chips[1].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
-      <div class="box black-square" id="e1" nextValidSquare="['d2', 'f2']"></div>
+
+      <div class="box black-square" id="e1" nextValidSquare="['d2', 'f2']">
+        <div class="piece black" :id="player1.chips[2].chipId"></div>
+      </div>
+
       <div class="box red-square"></div>
-      <div class="box black-square" id="g1" nextValidSquare="['f2', 'h2']"></div>
+
+      <div class="box black-square" id="g1" nextValidSquare="['f2', 'h2']">
+        <div class="piece black" :id="player1.chips[3].chipId"></div>
+      </div>
+
```

Comment:
here I've added the first 4 chips from player1. I am using the colon at the beginning of the `id` attribute to indicate that I want to bind my Vue data to the attribute. The colon there is shorthand for v-binding, and lets the compiler know that the content in the quotes, `player1.chips[3].chipId` is a Vue data variable, much like the *mustache syntax* above. VueJS lets us do some fun things with HTML tag attributes, but HTML attributes don't like mustache syntax, so there is this v-binding shorthand to work with.

If you checkout this branch and load this file in your browser, then inspect the HTML here, you will see that the individual black chips have ids like `black-1`, `black-2`, etc.